src/main/java/reposense/report/ReportGenerator.java

```diff
@@ -478,10 +486,35 @@ private static void handleFailedConfigs(
         return generatedFiles;
     }
 
+    /**
+     * Creates the .git-blame-ignore-revs file containing the contents of {@code IgnoreCommitList}
+     * in the config's repo root directory.
+     */
+    private static void generateIgnoreRevsFile(RepoConfiguration config) {
```

Comment:
Two questions:
1. Will this function override the existing `.git-blame-ignore-revs` in the repository if it already exists?
2. Will this generated file be deleted after the report generation is completed?

Reply:
1. Yes, it will override the existing file if it already exists.
2. Yes, the whole repo will be deleted therefore the file also gets deleted.